samples/core/src/main/java/io/oasp/gastronomy/restaurant/general/configuration/BaseWebSecurityConfig.java

```diff
@@ -148,14 +152,25 @@ protected JsonUsernamePasswordAuthenticationFilter getSimpleRestAuthenticationFi
   @PostConstruct
   public void init() throws Exception {
 
-    this.authenticationManagerBuilder.inMemoryAuthentication() //
-        .withUser("waiter").password("waiter").roles("Waiter").and() //
-        .withUser("cook").password("cook").roles("Cook").and() //
-        .withUser("barkeeper").password("barkeeper").roles("Barkeeeper") //
-        .and().withUser("chief").password("chief").roles("Chief");
+    this.authenticationManagerBuilder.inMemoryAuthentication(); //
+
+  }
+
+  @Override
+  protected UserDetailsService userDetailsService() {
+
+    return this.userDetailsService;
+  }
+
+  @Bean
+  public InMemoryUserDetailsManager inMemoryUserDetailsManager() {
 
-    // add our own authenticatonProvider that has add on functionality compared to spring security
-    this.authenticationManagerBuilder.authenticationProvider(this.authenticationProvider);
+    final Properties users = new Properties();
+    users.put("waiter", "waiter,Waiter");
```

Comment:
IMHO sick API. So it is "<password>,<Role>"? Better use explicit constructor with `Collection<UserDetails>`.